Loading checkpoint shards: 100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 2/2 [00:18<00:00,  9.43s/it]
Some weights of Qwen2ForSequenceClassification were not initialized from the model checkpoint at /home/exouser/Desktop/DeepSeek-R1-Distill-Qwen-7B and are newly initialized: ['score.weight']
You should probably TRAIN this model on a down-stream task to be able to use it for predictions and inference.
Device set to use cuda:0
0it [00:00, ?it/s]
Traceback (most recent call last):
  File "/home/exouser/Desktop/20_Advantage/Experiment.py", line 207, in <module>
    for epoch, batch in tqdm(enumerate(ppo_trainer.dataloader)):
  File "/home/exouser/miniconda3/envs/PPO/lib/python3.9/site-packages/tqdm/std.py", line 1181, in __iter__
    for obj in iterable:
  File "/home/exouser/miniconda3/envs/PPO/lib/python3.9/site-packages/accelerate/data_loader.py", line 563, in __iter__
    current_batch = next(dataloader_iter)
  File "/home/exouser/miniconda3/envs/PPO/lib/python3.9/site-packages/torch/utils/data/dataloader.py", line 701, in __next__
    data = self._next_data()
  File "/home/exouser/miniconda3/envs/PPO/lib/python3.9/site-packages/torch/utils/data/dataloader.py", line 757, in _next_data
    data = self._dataset_fetcher.fetch(index)  # may raise StopIteration
  File "/home/exouser/miniconda3/envs/PPO/lib/python3.9/site-packages/torch/utils/data/_utils/fetch.py", line 55, in fetch
    return self.collate_fn(data)
  File "/home/exouser/Desktop/20_Advantage/Experiment.py", line 118, in collator
    collated[key] = torch.stack([d[key] for d in data])
RuntimeError: stack expects each tensor to be equal size, but got [91] at entry 0 and [298] at entry 1
Traceback (most recent call last):
  File "/home/exouser/Desktop/20_Advantage/Experiment.py", line 207, in <module>
    for epoch, batch in tqdm(enumerate(ppo_trainer.dataloader)):
  File "/home/exouser/miniconda3/envs/PPO/lib/python3.9/site-packages/tqdm/std.py", line 1181, in __iter__
    for obj in iterable:
  File "/home/exouser/miniconda3/envs/PPO/lib/python3.9/site-packages/accelerate/data_loader.py", line 563, in __iter__
    current_batch = next(dataloader_iter)
  File "/home/exouser/miniconda3/envs/PPO/lib/python3.9/site-packages/torch/utils/data/dataloader.py", line 701, in __next__
    data = self._next_data()
  File "/home/exouser/miniconda3/envs/PPO/lib/python3.9/site-packages/torch/utils/data/dataloader.py", line 757, in _next_data
    data = self._dataset_fetcher.fetch(index)  # may raise StopIteration
  File "/home/exouser/miniconda3/envs/PPO/lib/python3.9/site-packages/torch/utils/data/_utils/fetch.py", line 55, in fetch
    return self.collate_fn(data)
  File "/home/exouser/Desktop/20_Advantage/Experiment.py", line 118, in collator
    collated[key] = torch.stack([d[key] for d in data])
RuntimeError: stack expects each tensor to be equal size, but got [91] at entry 0 and [298] at entry 1
